import_map.json

```diff
@@ -1,7 +1,4 @@
 {
-  "scopes": {
-    "THIS FILE EXISTS ONLY FOR VSCODE! IT IS NOT USED AT RUNTIME": {}
-  },
```

Comment:
This is still correct - the import map is not used to remap anything at runtime, as every Fresh project has it's own import map already.

Reply:
@lucacasonato 
> THIS FILE EXISTS ONLY FOR VSCODE!

It is written as above, but I think it is used in the lower part and `deno.json`, but is it necessary?
https://github.com/denoland/fresh/blob/4f671ec961c9497c2e2f7c055c0b8e2422b91b49/update.ts#L73

> IT IS NOT USED AT RUNTIME

Does this part only need?